lib/request.ts

```diff
@@ -135,15 +185,17 @@ export function getRequest({
 			}
 		}, initialPromise);
 
-	var interceptResponseOrError = async function (initialPromise) {
-		interceptors = exports.interceptors.slice().reverse();
-		return interceptors.reduce(function (promise, { response, responseError }) {
-			if (response != null || responseError != null) {
-				return promise.then(response, responseError);
-			} else {
-				return promise;
-			}
-		}, initialPromise);
+	const interceptResponseOrError = async function (initialPromise) {
+		return exports.interceptors
+			.slice()
+			.reverse()
+			.reduce(function (promise, { response, responseError }) {
+				if (response != null || responseError != null) {
+					return promise.then(response, responseError);
+				} else {
+					return promise;
+				}
+			}, initialPromise);
```

Comment:
If you're able to simplify this reduce promise chain building usign async/await I would love that

Reply:
Let me try that as a minor after we gather all the breaking changes.